ts_src/psbt.ts

```diff
@@ -831,16 +831,21 @@ export interface SignerAsync {
  */
 const transactionFromBuffer: TransactionFromBuffer = (
   buffer: Buffer,
-): ITransaction => new PsbtTransaction(buffer);
+): ITransaction => new PsbtTransaction({ buffer });
 
 /**
  * This class implements the Transaction interface from bip174 library.
  * It contains a bitcoinjs-lib Transaction object.
  */
-class PsbtTransaction implements ITransaction {
+export class PsbtTransaction implements ITransaction {
   tx: Transaction<bigint>;
-  constructor(buffer: Buffer = Buffer.from([2, 0, 0, 0, 0, 0, 0, 0, 0, 0])) {
-    this.tx = Transaction.fromBuffer<bigint>(buffer, undefined, 'bigint');
+  constructor({ tx, buffer }: { tx?: Transaction<bigint>; buffer?: Buffer }) {
```

Comment:
nit: minor change to avoid having to pass `{}` when you don't want to provide any args.

```suggestion
  constructor({ tx, buffer }: { tx?: Transaction<bigint>; buffer?: Buffer } = {}) {
```